src/input/checkbox/checkbox.jsx

```diff
@@ -3,15 +3,15 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import styles from '../../../src/input/checkbox/checkbox.styles.scss'
 
-const Checkbox = ({ style, className, input }) => {
+const Checkbox = ({ style, className, input, ...rest }) => {
   const classes = classnames(
     styles.checkbox,
     className,
-    { [styles.checked]: input.checked },
+    { [styles.checked]: input.checked, [styles.disabled]: rest.disabled },
```

Comment:
- You should extract disabled from `rest`.
 - If you are not doing this, the `onlyUpdateForPropTypes` will not work.
 - And I just look at it, but this component is not wrapped with this hoc `onlyUpdateForPropTypes`, can you add it maybe ?

Reply:
Ok, I'm on it